static/js/self-check.js

```diff
@@ -138,17 +137,34 @@ function loan() {
             var dueDate = new Date(data["due_date"]);
     		var dueDateText = (parseInt(dueDate.getMonth()) + 1) + "/" + dueDate.getDate() + "/" + dueDate.getFullYear();
     		$("#loanstable").append("<tr><td>" + data["title"] + "</td><td>" + dueDateText + "</td><td>" + data["item_barcode"] + "</td></tr>");
-    		
     		// write receipt and print, patron info found in login
-    		var receipt = window.open('','','width=200,height=100');
-    		receipt.document.write(
-    		"<font size='6'><b>Patron: </b>" + patron + "</font><br><font size='4'><b>Staff Status: </b>" + status + 
-    		"</font><br><b>Title: </b>" + data["title"] + 
-    		"<b><br>Author: </b>" + data["author"] + 
-    		"<br><b>Barcode: </b>" + data["item_barcode"] + 
-    		"<br><b>Due Date: </b>" + dueDateText);
+    		$.getScript("static/js/mustache.js", function(){

```

Comment:
again, you don't need to do this if you just include the mustache code in a <script> tag at the top of the file.